codecov.yml

```diff
@@ -0,0 +1,18 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+  coverage:
```

Comment:
Thanks, I'll fix that. I've copied this from https://github.com/codecov/support/blob/master/codecov.yml so you might want to fix that too?